bftengine/src/bftengine/ReplicaBase.hpp

```diff
@@ -128,6 +128,7 @@ class ReplicaBase {
   ///////////////////////////////////////////////////
   // ControlStateManger
   std::shared_ptr<ControlStateManager> controlStateManager_;
+  bool stopAtNextChecnpoint = false;
```

Comment:
`stopAtNextCheckpoint` and trailing underscore missing.

Reply:
fixed